fedimint-cli/src/client.rs

```diff
@@ -74,7 +74,11 @@ pub enum ClientCmd {
     /// Display wallet info (holdings, tiers)
     Info,
     /// Reissue notes received from a third party to avoid double spends
-    Reissue { oob_notes: OOBNotes },
+    Reissue {
+        oob_notes: OOBNotes,
+        #[arg(long = "no-wait", action = clap::ArgAction::SetFalse)]
```

Comment:
We'll need to conditionally handle this for back-compat

```
00:00:58 Error: command: /nix/store/fdbrlr3n4n94gs5a1lgnzskhgmv8pimq-fedimint-cli/bin/fedimint-cli --data-dir=/tmp/nix-shell.cTg2BC/mint_client_sanity-AneI/devimint-1686873-266/clients/receiver-0 reissue --no-wait AgEELaftFQBTAQEBsFhhsRLPTo/0Ad9oJmWF8RN3Qgx5x9k9JcI8wh6Md8H7gryAC6XrHjlx80dOG9BlWzWp9lJaw/qI+VGtcofUIP5jHzsmgEAX7OCNTy6cY/8=
00:00:58 
00:00:58 Caused by:
00:00:58     exit status: 2
00:00:58     stdout:
00:00:58     
00:00:58     stderr:
00:00:58     2024-05-01T09:03:57.056540Z DEBUG fedimint_cli: Starting fedimint-cli (version: a8422b84102ab5fc768307215d5b20d807143f27)
00:00:58     error: unexpected argument '--no-wait' found
```

https://github.com/fedimint/fedimint/actions/runs/8907459104/job/24461360848?pr=4922#step:5:1327

Reply:
Right. Will take care of it.